src/rt/sched/threadpool.h

```diff
@@ -82,13 +83,24 @@ namespace verona::rt
 
     ThreadState state;
 
+    /// Pool of cores shared by the scheduler threads.
+    CorePool<ThreadPool<T, E>, E>* core_pool = nullptr;
```

Comment:
Does this need to be a separate allocation?

Reply:
Same as above apparently.